src/helm/benchmark/static/schema_lite_v2.yaml

```diff
@@ -128,12 +133,30 @@ run_groups:
     description: Core Scenarios
     category: All scenarios
     subgroups:
-      - mmlu_pro
+      # - mmlu_pro
       - gpqa
+      - ifeval
+
+  # - name: mmlu_pro
+  #   display_name: MMLU-Pro
+  #   description: MMLU-Pro
+  #   metric_groups:
+  #     - accuracy
+  #     - efficiency
+  #     - general_information
+  #   environment:
+  #     main_name: exact_match  # non-CoT
+  #     main_split: test
+  #   taxonomy:
+  #     task: "?"
+  #     what: "?"
+  #     who: "?"
+  #     when: "?"
+  #     language: English
```

Comment:
Revert this change (otherwise it will conflict with the MMLU-Pro pull request)

Reply:
Makes sense. Both comments are addressed in the latest change.